expense-notifications-functions/src/index.ts

```
@@ -0,0 +1,129 @@
+);
+
+// Function to calculate the owed or payback amount for a member
+function calculateOwedAmount(expenseData: admin.firestore.DocumentData, memberId: string) {
+  // Get the total split amount for the member
+  const splitAmount = getTotalSplitAmountOf(expenseData, memberId);
+
+  // Get the amount paid by the member
+  const paidAmount = expenseData.paid_by[memberId] || 0;
+
+  // If the member has paid, calculate based on paid and split amounts
+  if (expenseData.paid_by.hasOwnProperty(memberId)) {
```

Biome

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)